packages/stacks/src/validation/memo-validation.ts

```diff
@@ -0,0 +1,9 @@
+import { MEMO_MAX_LENGTH_BYTES } from '@stacks/transactions';
+
+function exceedsMaxLengthBytes(string: string, maxLengthBytes: number): boolean {
+  return string ? Buffer.from(string).length > maxLengthBytes : false;
+}
```

Comment:
```suggestion
function exceedsMaxLengthBytes(value: string, maxLengthBytes: number): boolean {
  return value ? Buffer.from(value).length > maxLengthBytes : false;
}
```

something doesn't sit right about using a var name of a primitive type.